db/deploy/tables/form_data_status_type.sql

```diff
@@ -0,0 +1,28 @@
+-- Deploy ccbc:tables/form_data_status_type to pg
+
+begin;
+
+create table ccbc_public.form_data_status_type(
+  name varchar(1000) primary key,
+  description varchar(1000)
+);
+
+insert into ccbc_public.form_Data_status_type values
+ ('draft', 'Draft'), ('submitted', 'Submitted');
```

Comment:
Should we rename those to `pending` and `committed`, to avoid confusion with the application statuses?

Reply:
I think so, will also alleviate confusion of having a `withdrawn` application but a `submitted` form